### ISSUE TITLE
Problem with extended inner class when using kapt

### ISSUE THREAD

**Issue by janbina** [evernote/android-state#42]
I've got a problem with extended inner classes when using `kapt`. I have class `BasePresenter` with inner class like this:
```java
public abstract class BasePresenter {
    public class Inner {
        @State fields
    }
}
```
and extending class like this:
```java
public class SomePresenter extends BasePresenter {
    public class Inner extends BasePresenter.Inner {
        @State fields
    }
}
```
Base class StateSaver is generated with name `BasePresenter$Inner$$StateSaver`, extending class StateSaver looks like this: `SomePresenter$Inner$$StateSaver extends BasePresenter.Inner$$StateSaver` - there is dot instead of dollar sign after `BasePresenter`. 

When using `annotationProcessor`, it behaves as expected.

**Comment by vRallev** [comment]
I've seen some weird issue with kapt before that. I'll take a look.

**Comment by vRallev** [comment]
This should have been fixed in `1.2.1`. Please give this version a try and let me know if you still run into issues. Thanks
